Transaction.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_Assignment_2
+{
+    internal class Transaction
```

Comment:
Please remove the Transaction class and define the TransactionType enum at the namespace level.